terraform/main.tf

```diff
@@ -0,0 +1,174 @@
+terraform {
+  backend "s3" {
+    bucket = "home-terraform-1"
+    key    = "people-manager.tfstate"
+    region = "us-east-1"
+  }
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+    }
+  }
+}
+
+resource "aws_ecr_repository" "app_ecr_repo" {
+  name = "${var.project_name}-repo"
+}
+
+resource "aws_ecs_cluster" "my_cluster" {
+  name = "${var.project_name}-cluster"
+}
+
+resource "aws_ecs_task_definition" "app_task" {
+  family                   = "${var.project_name}-task"
+  container_definitions    = <<DEFINITION
+  [
+    {
+      "name": "${var.project_name}-task",
+      "image": "${aws_ecr_repository.app_ecr_repo.repository_url}",
+      "essential": true,
+      "portMappings": [
+        {
+          "containerPort": ${var.port_number},
+          "hostPort": ${var.port_number}
+        }
+      ],
+      "memory": 512,
+      "cpu": 256.
```

Comment:
I'm guessing this is a typo?

Reply:
that should def be a comma not a `.` but. It does seem to allocate `.25` cpu with this value 
<img width="1216" alt="Screenshot 2023-11-10 at 10 08 34 AM" src="https://github.com/campmindshark/people-manager/assets/3021938/e36058ff-1a5c-454f-af37-cc2e58c8b6dc">